eng/Versions.props

```diff
@@ -86,6 +86,7 @@
     <MicrosoftExtensionsHttpVersion>9.0.7</MicrosoftExtensionsHttpVersion>
     <SystemFormatsAsn1Version>9.0.7</SystemFormatsAsn1Version>
     <SystemTextJsonVersion>9.0.7</SystemTextJsonVersion>
+    <SystemThreadingChannelsVersion>9.0.7</SystemThreadingChannelsVersion>
```

Comment:
This should be added using dependency flow so that versions are automatically updated. This can be done using `darc add-dependency` but if you have any questions let me know and I can help.

Reply:
We will also need a version of it that is from the .NET 8 gen and a property in the property group below labeled LTS.